src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -116,10 +116,15 @@ def normalize_data(data_opm_reference, data_ensembles_flownet):
             tmp = np.append(tmp, data_ens, axis=1)
 
     matrix_data = np.append(data_opm_reference, tmp, axis=1)
-    scale = 1 / (
-        np.tile(data_opm_reference.max(), matrix_data.shape[0])
-        - np.tile(data_opm_reference.min(), matrix_data.shape[0])
-    )
+
+    if data_opm_reference.max() == data_opm_reference.min():
```

Comment:
I'd suggest to use `np.isclose()` to make sure this statement is evaluated to True also with minimal numerical differences.

```suggestion
    if np.isclose(data_opm_reference.max(), data_opm_reference.min()):
```